demo/presto/node.properties

```diff
@@ -0,0 +1,3 @@
+node.environment=docker
+node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
+node.data-dir=/var/presto/datanode.id=9e862deaf485
```

Comment:
should this datanode.id be on a new line?

Reply:
node.id will be added by presto when the container runs. I removed `node.id= 9e862deaf485` from the file now, we don't need it.

source: https://prestodb.io/docs/current/installation/deployment.html#node-properties